quickstarts/onepane/config.yml

```diff
@@ -2,34 +2,34 @@ id: e9b4adc3-a2e0-4177-b1f2-6b73254aa960
 
 description: |
   
-  ## OnePane
+  ## Onepane
 
-  OnePane acts as a central hub, bringing together information from various systems like Cloud, DevOps tools and Monitoring. 
+  Onepane acts as a central hub, bringing together information from various systems like Cloud, DevOps and Monitoring tools.
   This unified view lets you see how changes and events from these systems correlate with incidents, significantly speeding up the process of identifying the root cause (RCA).
 
-  ### Why OnePane?
-
-  By correlating updates in code, configurations, and deployments from various systems with New Relic incidents and entities, you can efficiently and quickly identify the exact root cause of an error. This way, you can see precisely which change triggered the problem, saving you a considerable amount of time in troubleshooting.
+  ### Why Onepane?
 
+  Onepane is a powerful addition to your existing New Relic, Cloud, and DevOps tools, providing seamless out-of-the-box integrations. It adopts an agentless approach and features automatic asset discovery, allowing you to get value in a matter of hours, not days or weeks. Onepane comprehends your unique landscape, generating a dynamic service map of your operational environment by harnessing data from various systems. This map assists your teams in swiftly resolving incidents and streamlines the audit process. Opt for Onepane to leverage additional ROI on your existing investments. 
 
   - Show changes and infrastructure events alongside errors to pinpoint potential root causes.
   - Enable root cause investigation by correlating incidents with change data and infrastructure data.
 
   ### Get started!
 
-  - To get started with OnePane, you can sign up for a [free cloud account](https://console.onepane.ai/).
-  - Integrate OnePane with New Relic to analyse deployment, change, and infrastructure event data to identify possible root causes. You can also review [additional documentation at OnePane](https://www.onepane.ai/docs/en/articles/8999270-adding-onepane-change-tracking-to-your-new-relic-deployment).
+  - To get started with Onepane, you can sign up for a [free SaaS account](https://console.onepane.ai/).
+  - You can access additional document on [Onepane documentation](https://www.onepane.ai/docs/en/articles/8999270-adding-onepane-change-tracking-to-your-new-relic-deployment).
+  - To know more about Onepane hosted plans or live demo book a meeting on [Onepane website](https://www.onepane.ai/about).
 
-summary: By correlating deployment change and infrastructure change from OnePane with New Relic, you can enable faster root cause identification for the incidents.
+summary: Onepane is an GenAI tool that helps you faster incident resolution with automated Root Cause Analysis. It uses data from New Relic, Cloud, and your DevOps stack to spot changes, events, and incidents. This means you can resolve incidents quicker and cut down your average resolution time (MTTR).
 
 level: Verified
 
```

Comment:
Somewhere in here you will need to reference the dashboard and the data sources so that it is related to this quickstart.

Reply:
Hi @jcountsNR, we are confused on how to add a reference to the data source while rewriting the summary to make it shorter. Any examples or documentation would be helpful.

Reply:
If you check the IO catalog and glance over the individual tiles tiles you should see what I was referring to. The text displayed there comes from the summary field. You can see the difference between `.NET core` and `.NET` as a good example. If it's too long it gets truncated and doesn't help understand what it is in a glance.

I would write something like 'Integrate your OnePane infrastructure with New Relic for key observability and root cause analysis` , or something short and simple.